UI/frontend-plugins/aja-output-ui/aja-ui-main.cpp

```diff
@@ -451,8 +450,10 @@ static void OBSEvent(enum obs_frontend_event event, void *)
 	}
 }
 
-static void aja_loaded(void *data OBS_UNUSED, calldata_t *calldata)
+static void aja_loaded(void *data, calldata_t *calldata)
```

Comment:
C++ file. Just remove var name..

Reply:
Unlike the other one, it's a static function (and callback) so there is no function declaration to get the name of the parameter, this is why I didn't just remove the name.

Reply:
How about `void */* data */`? (Or whatever clang-format refactors that to.)

Reply:
Should be good now